src/settings_gui.cpp

```diff
@@ -140,6 +142,46 @@ void ShowBaseSetTextfileWindow(TextfileType file_type, const TBaseSet* baseset,
 	new BaseSetTextfileWindow<TBaseSet>(file_type, baseset, content_type);
 }
 
+std::vector<int> _refresh_rates = { 30, 60, 75, 90, 100, 120, 144, 240 };
+
+/* We want 60Hz as the default for this pointer, though it gets overriden with whatever is in the config. */
+int *_current_refresh_rate = &_refresh_rates[1];
```

Comment:
Why is this a pointer?

Reply:
I considered it the best to not have a copy value of the current refresh rate but instead have a pointer inside of the refresh rates vector.

Reply:
Given we talk about an integer, that might be unexpected to most. Not talking about what happens if the vector relocates, etc (is that a thing with C++? :D).

Reply:
Except that that pointer becomes invalid as soon as you modify _refresh_rates It's used as an integer, just use `60` and don't worry about what the _refresh_rates vector contains

(also, "overridden")